osu.Game/OsuGame.cs

```diff
@@ -995,7 +995,10 @@ protected override void LoadComplete()
             }, topMostOverlayContent.Add);
 
             if (!args?.Any(a => a == @"--no-version-overlay") ?? true)
-                loadComponentSingleFile(versionManager = new VersionManager { Depth = int.MinValue }, ScreenContainer.Add);
+            {
+                dependencies.Cache(versionManager = new VersionManager { Depth = int.MinValue });
+                loadComponentSingleFile(versionManager, ScreenContainer.Add);
```

Comment:
This excursion is to prevent the version display on the bottom and the system title stepping on each other.